### PR TITLE
Schedule item component

### PR DESCRIPTION
**Pull request by marcol13**
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "next-i18next": "^12.0.1",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-icons": "^4.4.0",
```
[the remark this answers is not in the document]
Comment (marcol13) [reply package.json:25]: Thank you! I will change it 😅

File: src/components/ui/ScheduleItem/ScheduleItem.module.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+.textContainer {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  font-size: var(--font-size-m);
+  color: var(--color-white);
+}
+
+.lightBlue {
```
Comment (roertbb) [inline src/components/ui/ScheduleItem/ScheduleItem.module.scss:33]: maybe we could nest the color related classes inside `.item` to couple them together and prevent from reusing these classes in other context?

File: src/components/ui/ScheduleItem/ScheduleItem.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import classNames from "classnames";
+import styles from "./ScheduleItem.module.scss";
+import Icon from "@mdi/react";
```
Comment (roertbb) [inline src/components/ui/ScheduleItem/ScheduleItem.tsx:3]: We already have `Icon` component in `src/components/ui/Icon/Icon.tsx`. Can we decide on one implementation (the one from `components/ui` or `@mdi/react`) and re-export used icon paths in one file (for example `src/components/ui/Icon/index.ts`) to keep track on which icons are used and do our best to reduce number of used icons in order to make the views as consistent as possible?

Comment (marcol13) [reply src/components/ui/ScheduleItem/ScheduleItem.tsx:3]: Sorry, I am blind 🤦‍♂️ I forgot that we have special component for that - I will change it and remove `@mdi/react` from the project.

File: src/components/ui/ScheduleItem/ScheduleItem.module.scss [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  width: var(--icon-size);
+  height: var(--icon-size);
+  fill: currentcolor;
+}
+
+.textContainer {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  font-size: var(--font-size-m);
```
[the remark this answers is not in the document]
Comment (marcol13) [reply src/components/ui/ScheduleItem/ScheduleItem.module.scss]: Yes, sorry, my fault with that font-size. I don't see any visual difference between `space-around` and `space-between` in that case so I leave it as is 😀

**Review by roertbb — APPROVED** [review]

**Pull request merged**